app/scripts/modules/core/config/settings.js

```diff
@@ -2,5 +2,6 @@
 
 let angular = require('angular');
 
-module.exports = angular.module('spinnaker.core.config.settings', [])
+export const SETTINGS = 'spinnaker.core.config.settings';
```

Comment:
Doesn't look `SETTINGS` is being imported anywhere. In the interest of keeping the PR succinct, maybe revert this particular change?